src/ReactFinalForm.js

```diff
@@ -22,8 +22,9 @@ import isSyntheticEvent from './isSyntheticEvent'
 import type { FormRenderProps } from './types.js.flow'
 import ReactFinalFormContext from './context'
 import useLatest from './useLatest'
+import { version } from '../package.json'
 
-export const version = '6.2.0'
+export { version }
```

Comment:
Don't suppose you could do `export { version } from '../package.json'`, can you?

Reply:
I've started with that 😅 but you are using this variable when constructing `versions` object in this file and such reexport doesn't create a local binding which could be used.